include/types.h

```diff
@@ -40,6 +40,7 @@ struct Controller {
   /*0x0C*/ f32 stickMag;        // distance from center [0, 64]
   /*0x10*/ u16 buttonDown;
   /*0x12*/ u16 buttonPressed;
+           u16 buttonReleased;
```

Comment:
Change these comments to provide accurate data offsets:
```
  /*0x14*/ u16 buttonReleased;
  /*0x18*/ OSContStatus *statusData;
  /*0x1C*/ OSContPad *controllerData;
```

Reply:
Wouldn't it be
```
  /*0x14*/ u16 buttonReleased;
  /*0x16*/ OSContStatus *statusData;
  /*0x1A*/ OSContPad *controllerData;
```
since the new field is only 2 bytes?

Reply:
The next field is padded to 4 byte alignment because it's 4 bytes wide